nixos/modules/virtualisation/oci-common.nix

```diff
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.oci;
+in
+{
+  imports = [ ../profiles/qemu-guest.nix ];
```

Comment:
what does OCI use from this?

Reply:
The profile contains the necessary kernel modules for several QEMU/VirtIO devices. Running `lspci` on my instance running on OCI confirms that there are several emulated VirtIO devices present:
```
00:00.0 Host bridge: Red Hat, Inc. QEMU PCIe Host bridge
00:01.0 Display controller: Red Hat, Inc. Virtio GPU (rev 01)
00:02.0 USB controller: Red Hat, Inc. QEMU XHCI Host Controller (rev 01)
00:03.0 Ethernet controller: Red Hat, Inc. Virtio network device
00:10.0 PCI bridge: Red Hat, Inc. QEMU PCIe Root port
...
18:00.0 SCSI storage controller: Red Hat, Inc. Virtio SCSI (rev 01)
```